main.tf

```diff
@@ -92,3 +96,13 @@ module "cloud_connector" {
   tags       = var.tags
   depends_on = [module.org_cloudtrail, module.ecs_fargate_cluster]
 }
+
+module "cloud_bench" {
+  providers = {
+    aws = aws.member
+  }
+  source = "./modules/services/cloud-bench"
+
+  account_id  = var.org_cloudvision_member_account_id
```

Comment:
good simplification; let's start with this simple approach for the entry point to test e2e benchmark :+1: